src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -147,20 +164,18 @@ func runHyperKit(args []string) {
 	h.Memory = *mem
 	h.DiskSize = diskSz
 
-	// Add 9p and vsock for port forwarding if VPNKit is launched automatically
-	if *startVPNKit {
-		// The guest will use this 9P mount to configure which ports to forward
-		h.Sockets9P = []hyperkit.Socket9P{{Path: vpnKitPortSocket, Tag: "port"}}
-		// VSOCK port 62373 is used to pass traffic from host->guest
-		h.VSockPorts = append(h.VSockPorts, 62373)
-	}
-
 	err = h.Run(string(cmdline))
 	if err != nil {
 		log.Fatalf("Cannot run hyperkit: %v", err)
 	}
 }
 
+const (
+	networkingNone         string = "none"
+	networkingDockerForMac        = "docker-for-mac"
+	networkingVPNKit              = "vpnkit"
+)
+
```

Comment:
I don't know what's the idiomatic go way is but I find it odd to only define these constants after prominent use further up. I would define them after the import section at the beginning of the file...